### PR TITLE
HostingContentConfiguration fix

### PR DESCRIPTION
**Pull request by BjornRuud**
# Why?

Cells using HostingContentConfiguration could in some cases be blank before refreshed, and could have layout issues. This was caused by the child view controller being added to the parent too early so the parent view controller couldn't be found.

# What?

Manage child view controller after view has moved in/out of the parent view hierarchy.

# Version Change

Patch.

### CODE REVIEW

File: Demo/Sources/Cells/HostingConfigurationCell/HostingConfigurationCell.swift [hunk 1]
```diff
@@ -3,9 +3,12 @@ import FinniversKit
 import SwiftUI
 
 final class HostingContentConfigurationCell: UITableViewCell {
+    var onUpdateCellsConfiguredCount: (() -> Void)?
```
Comment (AtishPapa) [inline Demo/Sources/Cells/HostingConfigurationCell/HostingConfigurationCell.swift:6]: IMHO introducing this closure just for debug / demo purposes could lead to confusion (at least it did for me 😆). 

Wouldn't it be cleaner to make a specific subclass in the demo target? 🤔

Comment (BjornRuud) [reply Demo/Sources/Cells/HostingConfigurationCell/HostingConfigurationCell.swift:6]: It is a subclass in the demo target. 😄 But the name should include demo in it, it's a bit confusing.

Comment (AtishPapa) [reply Demo/Sources/Cells/HostingConfigurationCell/HostingConfigurationCell.swift:6]: Ohh man, I should get new glasses 👴 😆

**Review by robinsalehjan — APPROVED** [review]
LGTM 🥇

**Pull request merged**